content/booklet.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Digital Booklet"
+link-title: "booklet"
+draft: false
+type: page
+menu: main
+weight: 1
+---
+
+## Foreword
+
+![Constest Director Nadine](/nadine.webp)
```

Comment:
Can we add a bit of padding/margin around the side-picture? :slightly_smiling_face: The text is very close to the picture:

![image](https://github.com/user-attachments/assets/7b513eec-d8ea-4dba-8403-cbd52b79cfd8)